engines/agi/opcodes.cpp

```diff
@@ -462,6 +462,11 @@ void AgiEngine::setupOpCodes(uint16 version) {
 		}
 	}
 
+	// AGIMOUSE games use an modified push.script that updates mouse state
```

Comment:
```suggestion
	// AGIMOUSE games use a modified push.script that updates mouse state
```

Reply:
"Well, looks like we've got ourselves a reader!"